stylesheets/styles_about.css

```diff
@@ -0,0 +1,54 @@
+#prague {
+  background: url("../images/hipster-prague-cropped-a-lot.jpg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center center;
+  height: 304px;
+}
+
+#prague-banner {
+  margin-top: 150px;
+  margin-bottom: 0;
+}
+
+#seattle {
+  background: url("../images/seattle-dusk.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  height: 304px;
+}
+
+#seattle-banner {
+  margin-top: 150px;
+  color: white;
+}
+
+#future {
+  background: url("../images/silicon-valley.jpg");
+  background-size: cover;
```

Comment:
lines 29-32 are similiar to what's under #seattle.... Is there a way that you can think of to not duplicate code and dry this up?